### ISSUE TITLE
Started Erroring today

### ISSUE THREAD

**Issue by JoshAudibert** [imranismail/setup-kustomize#295]
I've been using this and it's worked until recently. The error I'm getting is 
```
Error: ENOENT: no such file or directory, chmod '/home/runner/work/_temp/5498170d-fce5-4d76-8267-c16d8194ccb4/kustomize'
```
Any help is appreciated.

**Comment by sachin-sanjay** [comment]
I've been getting the same error as well.

**Comment by rafaelgustavolv** [comment]
How to solve?

**Comment by edjo** [comment]
I'm having the same issue here

**Comment by magaldima** [comment]
seeing the same problem. successful at 2:04pm EST, and then started failing at 2:06pm EST.

**Comment by magaldima** [comment]
this might be related to the version of Kustomize v5.2.0. The team just released a patch v5.2.1 for this: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.2.1

**Comment by kfriend** [comment]
Confirmed working for me. @JoshAudibert you can probably close this
